### DISCUSSION TITLE
Run Focus_API using AMD CPU?

### DISCUSSION THREAD

**Discussion by MigueMartin99** [mrhan1993/Fooocus-API#382] [General]
I wanted to ask if there is a way to run Focus_API with an AMD GPU. Previously I was able to run the Focus model with an AMD GPU by editing the run.bat file and adding the lines:
```
.\python_embeded\python.exe -m pip uninstall torch torchvision torchaudio torchtext functorch xformers -y
.\python_embeded\python.exe -m pip install torch-directml
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --directml
pause
```

**Comment by mrhan1993** [comment]
Since I don't have an AMD GPU, I don't know if it will run on an AMD GPU.
